backend/controllers/attendanceStatistics.js

```diff
@@ -153,7 +153,9 @@ const attendanceStatisticsFactory = (oauthApi, elite2Api, whereaboutsApi, logErr
       const absences = attendances.filter(absence => absence.absentReason === reason)
 
       const offenderData = absences.map(absence => {
-        const offenderActivity = activities.find(activity => activity.bookingId === absence.bookingId)
+        const offenderActivity = activities.find(
+          activity => activity.bookingId === absence.bookingId && activity.eventId === absence.eventId
```

Comment:
guess your test data doesn't cover this specific case?

Reply:
No, I'll probably look at updating that in a later PR.